skewer.yaml

```diff
@@ -0,0 +1,141 @@
+      the **private1** cluster.
+    commands:
+      private1:
+        - run: mkdir pg-demo
+        - run: cd pg-demo
+        - run: git clone https://github.com/skupperproject/skupper-example-postgresql.git
+        - run: kubectl apply -f ~/pg-demo/skupper-example-postgresql/deployment-postgresql-svc.yaml
+          await: [deployment/postgresql]
+  - title: Create Skupper service for the Virtual Application Network
+    commands:
+      private1:
+        - run: skupper service create postgresql 5432
+      public1:
+        - run: skupper service status
+      public2:
+        - run: skupper service status
+    postamble: |
+      Note that the mapping for the service address defaults to `tcp`.
+  - title: Bind the Skupper service to the deployment target on the Virtual Application Network
+    commands:
+      private1:
+        - run: skupper service bind postgresql deployment postgresql
+      public1:
+        - run: skupper service status
+      public2:
+        - run: skupper service status
+    postamble: |
+      Note that the **private1** is the only cluster to provide a target.
+  - title: Create interactive pod with PostgreSQL client utilities
+    commands:
+      private1:
+        - run: "kubectl run pg-shell -i --tty --image quay.io/skupper/simple-pg --env=\"PGUSER=postgres\" --env=\"PGPASSWORD=skupper\" --env=\"PGHOST=$(kubectl get service postgresql -o=jsonpath='{.spec.clusterIP}')\" -- bash"
+      public1:
+        - run: "kubectl run pg-shell -i --tty --image quay.io/skupper/simple-pg --env=\"PGUSER=postgres\" --env=\"PGPASSWORD=skupper\" --env=\"PGHOST=$(kubectl get service postgresql -o=jsonpath='{.spec.clusterIP}')\" -- bash"
+      public2:
+        - run: "kubectl run pg-shell -i --tty --image quay.io/skupper/simple-pg --env=\"PGUSER=postgres\" --env=\"PGPASSWORD=skupper\" --env=\"PGHOST=$(kubectl get service postgresql -o=jsonpath='{.spec.clusterIP}')\" -- bash"
+    postamble: |
+      Note that if the session is ended, it can be resumed with the following:
+           ```bash
+           kubectl attach pg-shell -c pg-shell -i -t
+           ```
+  - title: Create a Database, Create a Table, Insert Values
+    preamble: |
+      Using the 'pg-shell' pod running on each cluster, operate on the database:
+    commands:
+      private1:
+        - run: createdb -e markets
+      public1:
+        - run: psql -d markets
+        - run: "create table if not exists product (id SERIAL, name VARCHAR(100) NOT NULL, sku CHAR(8));"
+      public2:
+        - run: psql -d markets
+        - run: "INSERT INTO product VALUES(DEFAULT, 'Apple, Fuji', '4131');"
+        - run: "INSERT INTO product VALUES(DEFAULT, 'Banana', '4011');"
+        - run: "INSERT INTO product VALUES(DEFAULT, 'Pear, Bartlett', '4214');"
+        - run: "INSERT INTO product VALUES(DEFAULT, 'Orange', '4056');"
+    postamble: |
+      From any cluster, access the `product` tables in the `markets` database to view contents.
+summary: |
```

Comment:
The summary is rendered at the end, so this text should either go back to the start (the overview) or be recast as "what you did", not what you will do.

Reply:
Thanks! Summary updated.